delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Resources.scala

```diff
@@ -21,14 +22,17 @@ import ch.epfl.bluebrain.nexus.delta.sdk.model.resources.ResourceRejection._
 import ch.epfl.bluebrain.nexus.delta.sdk.model.resources.ResourceState._
 import ch.epfl.bluebrain.nexus.delta.sdk.model.resources.{ResourceCommand, ResourceEvent, ResourceRejection, ResourceState}
 import ch.epfl.bluebrain.nexus.delta.sdk.model.schemas.Schema
+import com.typesafe.scalalogging.Logger
 import fs2.Stream
 import io.circe.Json
 import monix.bio.{IO, Task, UIO}
 
 /**
   * Operations pertaining to managing resources.
   */
-trait Resources {
+trait Resources extends EventResolver {
```

Comment:
Why `EventResolver` is only implemented here and not in `Resolvers`, `Schemas`, `Storages`, `BlazegraphViews`, `ElasticSearchViews`?

Reply:
I wanted to keep this PR small, the rest can be implemented in another PR.